src/helpers/git_helpers/remote_helpers.sh

```diff
@@ -18,3 +18,12 @@ function remote_repository_name {
   local domain=$(remote_domain)
   remote_url | sed -E "s#.*$domain[/:](.+)#\1#" | sed "s/\.git$//"
 }
+
```

Comment:
Please add one more empty line (the codebase is following the pattern of two empty lines between functions)